browser-test/src/admin/northstar_admin_program_preview.test.ts

```diff
@@ -0,0 +1,51 @@
+import {test} from '../support/civiform_fixtures'
+import {
+  enableFeatureFlag,
+  loginAsAdmin,
+  validateScreenshot,
+  waitForPageJsLoad,
+} from '../support'
+
+test.describe('admin program preview', {tag: ['@northstar']}, () => {
+  test.beforeEach(async ({page}) => {
+    await enableFeatureFlag(page, 'north_star_applicant_ui')
+  })
+
+  test('preview program and use back button', async ({
+    page,
+    adminPrograms,
+    adminQuestions,
+    applicantQuestions,
+  }) => {
+    await loginAsAdmin(page)
+
+    await adminQuestions.addEmailQuestion({questionName: 'email-q'})
```

Comment:
suggestion (non-blocking): use `test.step` to break these tests up into logical chunks: https://github.com/civiform/civiform/wiki/Testing#steps